website/content/docs/what-is-vault.mdx

```diff
@@ -18,8 +18,22 @@ difficult and platform-specific. Adding on key rolling, secure storage, and
 detailed audit logs is almost impossible without a custom solution. This is
 where Vault steps in.
 
-Examples work best to showcase Vault. Please see the
-[use cases](/docs/use-cases).
+### How does Vault work?
+
+![How Vault Works](/img/how-vault-works.png)
+
+Vault validates, authorizes, and provides clients (users, machines, apps) access to secrets or stored sensitive data. The core Vault worklflow consists of four stages:
+
+![Vault Workflow](/img/vault-workflow-diagram1.png)
```

Comment:
I actually wonder if you need to cut either the graphic or the list below because they say the same things. In fact, I would prefer to keep the text because it would be easier to edit.

Reply:
Ok, I can see where it may appear to be duplicate info. I was thinking of providing a bit more detail in the bulleted list. Basically, add more context to it. I think the visual gives user a nice quick visual of the workflow and they can refer to the bulleted list for more info. @laurapacilio , thoughts on this?

Reply:
hmmm So I think to make things more accessible, we should probably repeat the workflow in the text and use the bullets to add information that the picture can't capture. The picture should be reinforcing the text. My opinion is that users shouldn't HAVE to even look at the picture to understand the point, but that the picture should kind of....illustrate the points further. Like bonus content :) 

The way we handled this in Terraform docs is we kind of listed out the workflow item int he written bullet and tried to tie it to a real-world example. Whereas the picture just reinforces the workflow and tells the user something about how the Terraform project is organized/communicates with Terraform/the providers. You don't have to do it exactly the same as us! But I'm wondering if you could:
- Start with the list. 
- Rather than repeating the text in the graphic for each stage, could you expand? Give us an understanding of what that actually MEANS in practice. "Vault provides access to secrets, keys, and encryption capabilities based on policies associated with the client’s identity." Could you add like "for example...blah blah". 

This way it's not repetitive. Each has a purpose!